### PR TITLE
gui: locale: Ensure consistent branding

### PR DESCRIPTION
**Pull request by mdhorn**
All occurrences of Clear Linux* OS should be the same.

### CODE REVIEW

File: locale/es_MX/LC_MESSAGES/clr-installer.po [hunk 1]
```diff
@@ -398,11 +398,11 @@ msgstr "MX: Confirm"
 msgid "You can use alpha-numeric characters and hyphens with a maximum of %d characters"
 msgstr "MX: You can use alpha-numeric characters and hyphens with a maximum of %d characters"
 
-msgid "Welcome to Clear Linux* Desktop Installation"
+msgid "Welcome to Clear Linux* OS Desktop Installation"
 msgstr "Bienvenido a la instalación de Clear Linux* para computadoras de escritorio"
```
Comment (reaganlo) [inline locale/es_MX/LC_MESSAGES/clr-installer.po]: I know that the final translation will fix this, but for the time being can you please modify the translation to:

msgstr "Bienvenido a la instalación de **operativo** Clear Linux* para computadoras de escritorio"

File: locale/es_MX/LC_MESSAGES/clr-installer.po [hunk 2]
```diff
@@ -398,11 +398,11 @@ msgstr "MX: Confirm"
...
 
 #, c-format
-msgid "Thank you for choosing Clear Linux*"
+msgid "Thank you for choosing Clear Linux* OS"
 msgstr Gracias por elegir Clear Linux*"
```
Comment (reaganlo) [inline locale/es_MX/LC_MESSAGES/clr-installer.po]: I know that the final translation will fix this, but for the time being can you please modify the translation to:

msgstr "Gracias por elegir **operativo** Clear Linux*"

File: locale/zh_CN/LC_MESSAGES/clr-installer.po [hunk 3]
```diff
@@ -397,11 +397,11 @@ msgstr "CN: Confirm"
 msgid "You can use alpha-numeric characters and hyphens with a maximum of %d characters"
 msgstr "CN: You can use alpha-numeric characters and hyphens with a maximum of %d characters"
 
-msgid "Welcome to Clear Linux* Desktop Installation"
+msgid "Welcome to Clear Linux* OS Desktop Installation"
 msgstr "欢迎 安装 Clear Linux* 桌面"
```
Comment (reaganlo) [inline locale/zh_CN/LC_MESSAGES/clr-installer.po]: I know that the final translation will fix this, but for the time being can you please modify the translation to:

msgstr "欢迎 安装 Clear Linux* **OS** 桌面"

File: locale/zh_CN/LC_MESSAGES/clr-installer.po [hunk 4]
```diff
@@ -397,11 +397,11 @@ msgstr "CN: Confirm"
...
 
 #, c-format
-msgid "Thank you for choosing Clear Linux*"
+msgid "Thank you for choosing Clear Linux* OS"
 msgstr "感谢您选择 Clear Linux*"
```
Comment (reaganlo) [inline locale/zh_CN/LC_MESSAGES/clr-installer.po]: I know that the final translation will fix this, but for the time being can you please modify the translation to:

msgstr "感谢您选择 Clear Linux* **OS**"

**Comment by mdhorn** [comment]
@reaganlo Made the suggest updates, but please verify.

**Comment by reaganlo** [comment]
LGTM. Thanks for making the change. Approved.

**Pull request merged**